ConvivaExampleApp/src/main/java/com/bitmovin/analytics/convivaanalyticsexample/MainActivity.java

```diff
@@ -135,11 +136,17 @@ private AdvertisingConfiguration buildAdConfiguration() {
     protected void onResume() {
         super.onResume();
         bitmovinPlayerView.onResume();
+        try {
+            convivaAnalytics.initializeSession();
+        } catch (ConvivaAnalyticsException e) {
+            e.printStackTrace();
+        }
     }
 
     @Override
     protected void onPause() {
-        bitmovinPlayerView.onPause();
+        bitmovinPlayerView.onStop();
```

Comment:
@irishsheila Would this also fire on a standard pause (clicking the player to pause it) as well? We wouldn't want to end the session in that case.

Reply:
No, onPause isn't called on a standard pause (pressing pause or play). It is only called when the app is backgrounded (I did double check this). Same with onResume. Only when the app is backgrounded.